src/ajax.js

```diff
@@ -13,6 +13,7 @@ define([
 
 var
 	rhash = /#.*$/,
+	rextraQuestions = /\?+/,
```

Comment:
Perhaps `/\?\?+/` would be faster as in this PR you're currently changing single question marks to themselves.